src/components/markdownRenderer/markdown-renderer.module.css

```diff
@@ -7,6 +7,7 @@
         #7cf952 55%
     );
     text-decoration: underline;
+    text-decoration-skip-ink: none;
     text-decoration-thickness: 19px;
     text-decoration-color: #7cf952;
     display: inline;
```

Comment:
To fix the h1 issue you could do:
.markdown h1 {
    font-size: 48px;
    color: black;
    text-decoration: underline;
    text-decoration-skip-ink: none;
    text-decoration-thickness: 50px;
    text-underline-offset: -40px;
    text-decoration-color: #7cf952;
}

Reply:
@ChinemeremChigbo take a look when you're free! Thanks for the suggestion :)